sections/product-main-template.liquid

```diff
@@ -285,18 +288,26 @@
     <li class="tab-options"><a href="#tab-3">Reviews</a></li>
     <li class="tab-options"><a href="#tab-4">Questions & Answers</a></li>
   </ul>
-  <hr style="margin-top:0px;">
+  <hr class="hr-no-margin">
   <div id="tab-1">
     {% include "product-features" %}
   </div>
   <div id="tab-2">
     {% include "product-specs" %}
   </div>
   <div id="tab-3">
+    <div class="tabs-margin-div">
```

Comment:
I want to really get our styles and especially typography under control so let's move this into the main style file so we can easily see all the variants we have in play.